server/routes/api/v1Campaigns.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express')
+const router = express.Router()
+const Campaign = require('../../db/models/campaign')
+
+router.get('/:id', async (req, res) => {
+  try {
+    const campaign = await Campaign.query().findById(req.params.id)
+    if (campaign) {
+      res.send(campaign)
```

Comment:
This is a small nitpick, but I would prefer that we use the following pattern for responses: `return res.status(<status code>).json({ campaign }).end()`. That way we are explicitly adding each piece of the response. Returning the response will prevent code in the route from running after Express sends the response. The `end()` will close the http response which prevents hanging requests in some cases.

Reply:
Thanks for the guidance. Going to make the responses more explicit and will implement `return` and `end()` for better request handling.